Aptoide/AppCoins-Integration-AndroidStudio-Plugin
# Update of method snippets and text

To pass from 0.6.x version to 0.7.x version it was added tge AppCoinsBillingClient and the CheckPurchases all the remaining code keeps updated with the Catappult documentation

**What does this PR do?**

This PR updates the Android Studio Plugin from version 0.6.7.0 to 0.7.1.0

For that it was necessary to change:
Snippets (Java / Kotlin) (where all code snippets are placed)
Dialogs (where all text is set)
CardLayout (where the UI is displayed)
plugin.xml to update the version and MetricsClient that was giving an error on an import which lib was not being found neither used

**How should this be manually tested?**

1) To test this update you will need to run the Gradle tasks, the Build > buildDependents 
2) This will generate a .zip file on folder build/distributions
3) You will open Android Studio and import the .zip file - this is done by Setting > Plugins clique on the gear icon beside Marketplace | Installed and pick option Import plugin from file (it may be useful to delete the plugin firstly if installed) 
4) It should display the changes on Starting Service Connection - AppBillingClient and CheckPurchase

**What are the relevant tickets?**

Tickets related to this pull-request: 
- https://aptoide.atlassian.net/jira/software/c/projects/APT/issues/APT-3880

**Questions:**

-

**Code Review Checklist**
